source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -247,6 +253,21 @@ The following example performs these actions:
    `Dispose() <https://learn.microsoft.com/en-us/dotnet/api/system.idisposable.dispose?view=net-7.0>`__
    method once the cursor is no longer in use.
 
+.. _csharp-retrieve-instruqt-lab:
+
+Complete an Interactive Lesson
+------------------------------
+
+This lab helps you understand how to perform read operations in MongoDB
+by using the ``Find()`` method. You can complete this lab directly in
+your browser window without installing MongoDB or a code editor.
+
+.. tip::
+   
+   To expand the lab to a full-screen format, click the full-screen
+   button, :guilabel:`⛶`, in the lower-right corner of the lab pane.
```

Comment:
Suggestion:

I think this could be more clear by describing the position as follows, but no strong opinion.

```suggestion
   button, :guilabel:`⛶`, in the bottom right corner of the lab pane.
```